src/Sleep.js

```diff
@@ -0,0 +1,34 @@
+class Sleep {
+  constructor(sleepData) {
+    this.userSleepData = sleepData
+  }
+
+  averageHoursOfSleep(id){
+    const userSleep = this.userSleepData.filter((user) => user.userID === id);
+      let sum = 0;
+        userSleep.forEach((user) => {sum += user.hoursSlept});
+          return Number((sum / userSleep.length).toFixed(1));
+  }
+
+  averageQualityOfSleep(id){
+    const userSleep = this.userSleepData.filter((user) => user.userID === id);
+      let sum = 0;
+        userSleep.forEach((user) => {sum += user.sleepQuality});
```

Comment:
Would we want to do reduce? Let's table this idea for refractor though if this is running!